id_infrastructure/firestore_uuid_table.py

```diff
@@ -0,0 +1,109 @@
+import json
+
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import firestore
+
+import uuid
+
+BATCH_SIZE = 500
+
+class FirestoreUuidTable(object):
+    def __init__(self, table_name, crypto_token_path, uuid_prefix):
+        cred = credentials.Certificate(crypto_token_path)
+        firebase_admin.initialize_app(cred)
+        self._client = firestore.client()
+        self._table_name = table_name
+        self._uuid_prefix = uuid_prefix
+
+    def data_to_uuid_batch(self, list_of_data_requested):
```

Comment:
After just reading this code I don't have a good sense as to what "data" is. The PR message adds a bit more context but once we merge that gets lost, so could you copy and paste the PR message at the top of this file?

Reply:
Good idea - added thanks